index.html

```diff
@@ -14,24 +14,28 @@
   </script>
   <script class="remove" src="js/config.js">
   </script>
-  <script>
+  <script class="remove">
    respecConfig = {...organisationConfig, ...respecConfig};
+   respecConfig.localBiblio = {...organisationConfig.localBiblio, ...respecConfig.localBiblio};
```

Comment:
Meestal houden we het bij de gedeelde lijst, maar dit is wel een nettere oplossing dan ik hier heb voorgesteld:
https://github.com/Logius-standaarden/respec/issues/52#issuecomment-1733721606